app/controllers/itsi_login_controller.rb

```diff
@@ -0,0 +1,50 @@
+class ItsiLoginController < ApplicationController
```

Comment:
This isn't specific to ITSI it should be called `ImportedLoginController` instead. And other uses of ITSI in this file should be replaced with `imported` like `imported_user_authentication`, and `User.verified_imported_user`